src/components/ResoucesList.tsx

```diff
@@ -0,0 +1,242 @@
+import { format } from "date-fns";
+import { Link } from "react-router-dom";
+import "./ResourcesList.css";
+import axios from "axios";
+import { BackendURL } from "../utils/backendURL";
+import useFetchResources from "../utils/useFetchResources";
+import { useState } from "react";
+
+function textSummary(text: string, length: number) {
+  let returnText = "";
+
+  if (text.length > length) {
+    returnText = text.slice(0, length) + "...";
+  } else {
+    returnText = text;
+  }
+  return returnText;
+}
+
+export function ResourcesList(): JSX.Element {
+  const { resources, updateResources } = useFetchResources();
+  const [searchValue, setSearchValue] = useState("");
```

Comment:
We would suggest adding the types here so that we know what we are talking about.